snyk-monitor/templates/deployment.yaml

```diff
@@ -20,6 +20,13 @@ spec:
     spec:
       serviceAccountName: {{ include "snyk-monitor.name" . }}
       restartPolicy: Always
+      initContainers:
+        - name: volume-permissions
+          image: busybox
```

Comment:
This must be configurable actually, otherwise it won't work for on-prem air-gapped deployments (meaning without internet access, with some local container Registry where users have only approved images that they allow on the cluster) 🙏

Reply:
Ah, I hadn't thought of that - great point! Will add and document an override 👍

Reply:
I guess this means that the initContainer should mount the `docker-config` volume too, if I'm right in thinking that this is where it gets private registry credentials from?

Reply:
I don't think so, the `docker-config` volume is for `kubernetes-monitor` to be able to pull and analyze images from private registries. This image here is a request from Kubernetes itself to deploy it for us

Reply:
Aha ok. So if customer wants to use a local container registry to get an alternative image to busybox, do they need any new way to send credentials, or is this already taken care of somewhere?

Reply:
They will configure it themselves yep! 🙏